server/app.js

```diff
@@ -128,17 +129,17 @@ module.exports = function createApp({
   const cacheControl = { maxAge: config.staticResourceCacheDuration * 1000 }
 
   ;[
-    '../assets',
-    '../assets/stylesheets',
-    '../assets/js',
-    '../node_modules/govuk-frontend/govuk/assets',
-    '../node_modules/govuk-frontend',
-    '../node_modules/@ministryofjustice/frontend/',
+    '/assets',
+    '/assets/stylesheets',
+    '/assets/js',
+    `/node_modules/govuk-frontend/govuk/assets`,
+    `/node_modules/govuk-frontend`,
+    `/node_modules/@ministryofjustice/frontend/`,
   ].forEach(dir => {
-    app.use('/assets', express.static(path.join(__dirname, dir), cacheControl))
+    app.use('/assets', express.static(path.join(process.cwd(), dir), cacheControl))
```

Comment:
need to work both when deployed and locally
`__dirname` shows location of this file whereas cwd is the directory where node is run from.

Using `__dirname`, locally when code is compiled to a `dist` folder, node_modules is `../../` away whereas when running in docker its `../`

With `cwd` it's alway at the top level